src/index.css

```
@@ -1,5 +1,30 @@
 @import-normalize;
 
+* {
+  box-sizing: border-box;
+}
+
+li {
+  list-style-type: none;
+}
+
+table,
+th,
+td {
+  border: 1px solid black;
+  border-collapse: collapse;
+  text-align: center;
+}
+table {
+  width: 100%;
+}
+thead {
+  background-color: #e5e5e5;
+}
+tbody {
+  background-color: #fff;
```

The same as always. You should use css variables.

---

Ok, I'll try not to break it again.
```suggestion
  thead {
  background-color: var(--bg-color-secondery);
}
tbody {
  background-color: var(--color-primary);
}
```